Source/WebCore/platform/PlatformMouseEvent.h

```diff
@@ -68,6 +68,8 @@ class PlatformMouseEvent : public PlatformEvent {
     const IntPoint& position() const { return m_position; }
     const IntPoint& globalPosition() const { return m_globalPosition; }
     const IntPoint& movementDelta() const { return m_movementDelta; }
+    // Unaccelerated pointer movement
+    const IntPoint& unadjustedMovementDelta() const { return m_unadjustedMovementDelta; }
```

Comment:
I know you're following `movementDelta`, but a delta would normally be an `IntSize`. The right way to fix this would be to do a first PR that changes `movementDelta` to an IntSize, then do this PR using IntSize.

Reply:
Why IntSize? From a quick grep of the code, I see most usages of IntSize as representing actual sizes (i.e. things that have area) rather than point deltas. While the underlying representation of IntSize is the same as IntPoint and can hold negative width/height, my intuition is that a size is not generally thought of as being negative, whereas the idea of a negative point is common. I suppose this is why CoreGraphics defines CGVector, even though it is rarely used.

To be clear, I'm happy to change it if that's the way the WebKit team wants it, but just as an outsider it's not clear to me how IntSize improves clarity or utility here.

Reply:
Generally we use `IntSize` to represent a delta (or vector), and `IntPoint` to represent a geometric point. But OK to follow the other data members for now.